parser.ts

```diff
@@ -197,6 +197,21 @@ export function traverseExpr(c : TreeCursor, s : string) : Expr<SourceLocation>
         obj: objExpr,
         field: propName
       }
+    case "TupleExpression":
+      let tuple :Expr<SourceLocation>[] = [];
+      c.firstChild(); // Open parenthesis "("
+      c.nextSibling();
```

Comment:
Again merge conflict warning, please talk to the sets/tuples team of compiler B

Reply:
This has been removed.